raft-tech/TANF-app
# Feat/1788 resource cards

## Summary of Changes
_Provide a brief summary of changes_
Pull request closes #1788 

## How to Test
This worked on my Linux system, but the fontend docker-compose command fails on mac with the following error:
<details>

 > [localdev 6/7] RUN npm ci:                                                   
#0 98.84 npm notice                                                             
#0 98.84 npm notice New minor version of npm available! 8.1.2 -> 8.13.2         
#0 98.84 npm notice Changelog: <https://github.com/npm/cli/releases/tag/v8.13.2>
#0 98.84 npm notice Run `npm install -g npm@8.13.2` to update!                  
#0 98.84 npm notice 
#0 98.84 npm ERR! code 1
#0 98.84 npm ERR! path /home/node/app/node_modules/puppeteer
#0 98.84 npm ERR! command failed
#0 98.84 npm ERR! command sh -c node install.js
#0 98.84 npm ERR! The chromium binary is not available for arm64.
#0 98.84 npm ERR! If you are on Ubuntu, you can install with: 
#0 98.84 npm ERR! 
#0 98.84 npm ERR!  sudo apt install chromium
#0 98.84 npm ERR! 
#0 98.84 npm ERR! 
#0 98.84 npm ERR!  sudo apt install chromium-browser
#0 98.84 npm ERR! 
#0 98.84 npm ERR! /home/node/app/node_modules/puppeteer/lib/cjs/puppeteer/node/BrowserFetcher.js:115
#0 98.84 npm ERR!                     throw new Error();
#0 98.84 npm ERR!                     ^
#0 98.84 npm ERR! 
#0 98.84 npm ERR! Error
#0 98.84 npm ERR!     at /home/node/app/node_modules/puppeteer/lib/cjs/puppeteer/node/BrowserFetcher.js:115:27
#0 98.84 npm ERR!     at FSReqCallback.oncomplete (node:fs:198:21)
#0 98.96 
#0 98.96 npm ERR! A complete log of this run can be found in:
#0 98.96 npm ERR!     /home/node/.npm/_logs/2022-07-06T18_04_53_137Z-debug.log
------
failed to solve: executor failed running [/bin/sh -c npm ci]: exit code: 1

</details>

```
cd tdrs-frontend && docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d
cd tdrs-backend && docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d 
```

On mac I start the backend with 
```
cd tdrs-frontend && docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d
```

And the frontend with 
```
npm start
```

1. Open http://localhost:3000/.


## Deliverables


**Acceptance Criteria:** 

- [x] Unauthenticated landing page no longer has empty "resources"
- [x] Page matches Figma wireframes

**Tasks:**

- [x] Using the USWDS Card Component, create the dynamic cards for these resource links.
- [x] Ensure links within Figma are attached to the corresponding cards and functional



### [Deliverable 2: Tested Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-2-Tested-Code)

+ Are all areas of code introduced in this PR meaningfully tested?
+ Are code coverage minimums met?
  + [x] Frontend coverage: 94.97%
  + [x] Backend coverage: 94.97%

### [Deliverable 3: Properly Styled Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-3-Properly-Styled-Code)


### [Deliverable 4: Accessible](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-4-Accessibility)

@ttran-hub noted some minor things [here](https://github.com/raft-tech/TANF-app/pull/1924#issuecomment-1192027460) to be addressed in a follow-up ticket. 

### [Deliverable 5: Deployed](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-5-Deployed)


### [Deliverable 6: Documented](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-6-Code-documentation)


### [Deliverable 7: Secure](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-7-Secure)


### [Deliverable 8: User Research](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-8-User-Research)

Research product(s) clearly articulate(s):
